packaging/debian/postrm

```diff
@@ -7,11 +7,12 @@ if [ -f /usr/share/debconf/confmodule ]; then
 fi
 
 case "$1" in
-	purge | remove)
+	remove)
+	systemctl daemon-reload
 	groupdel percona-telemetry || true
 	;;
 
-	upgrade | failed-upgrade | abort-install | abort-upgrade | disappear) ;;
+	purge | upgrade | failed-upgrade | abort-install | abort-upgrade | disappear) ;;
```

Comment:
Why you moved purge action here?

Reply:
I noticed that when deleting the telemetry-packages with the "apt purge percona-telemetry-agent" command, the remove condition is executed first and then purge, as a result of which the `groupdel percona-telemetry ` was being executed twice which gave a warning/error about missing percona-telemetry group. It works fine now after moving the purge action.